highfidelity/hifi
# Move blendshape packing to worker thread

Also, parallelize the initial packing and adds a menu option for enabling/disabling blendshapes.

Test plan:
- Go to a domain with someone else.  Talk and look at each other.  The blendshapes should work as in master (eye blinking, mouth moving, etc.).
- Change avatars.  Blendshapes should still work.
- Play with the Developer -> Render -> Compute Blendshapes option.  When it's enabled, you should see blendshapes on yourself and other avatars working as expected.  When it's off, blendshapes should stop updating, but everything should still render correctly.
